src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml

```diff
@@ -17,6 +17,14 @@ limitations under the License.
 <configurator>
   <domain name="General"/>
 
+  <enum name="DlStartUpOnOff" type="enum8">
```

Comment:
Why "Dl" at the start of the name?

But also, the spec does not in fact define a named enum here.  The current cluster XML matches the spec; why are we changing them to not match?

Reply:
"Dl" should have been "OnOff". Will update.

@bzbarsky-apple , please see https://csamembers.slack.com/archives/CUXQ5QRMK/p1644616505533729.

The problem with the [src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml](https://github.com/project-chip/connectedhomeip/pull/16466/files/28abb25dc5ba366308d4f5261581b0ce7c8e3f12#diff-3cd6e11dda9d6cc85e14bd95b76c8d57cba5d758ee168a28464633ac3a2018c6) is that the START_UP_ON_OFF attribute lost its enumerated values. 

In the Zigbeee ZCL, the attribute had its enumerated values builtin into the attribute, whereas for matter, the onoff-cluster.xml file removed the relationship.

**Zigbee ZCL OnOff Cluster, StartUpOnOff attribute:**
`      <attribute id="4003" name="StartUpOnOff" type="enum8" writable="true">
        <restriction>
          <type:enumeration value="00" name="SetOnOffTo0" />
          <type:enumeration value="01" name="SetOnOffTo1" />
          <type:enumeration value="02" name="TogglePreviousOnOff" />
          <type:enumeration value="FF" name="SetPreviousOnOff" />
        </restriction>
      </attribute>`

**Matter ZCL OnOff Cluster, START_UP_ON_OFF attribute:**
 `    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="enum8"               writable="true" optional="true">StartUpOnOff</attribute>`

The application needs to expose these possible enumerated values for this attribute.

Reply:
@hubTab Sounds like we need a spec issue and a spec update, no?

Reply:
Thanks @bzbarsky-apple. Indeed, will create a spec issue and will update the spec, too.

Reply:
@hubTab Drop a link to that PR in the dm-light-n-occ and/or chip-spec-writing channels and ask for reviews?  But yes, so reviews get auto-requested....